js/classes/entities/midground/chad.js

```diff
@@ -142,6 +142,16 @@ class Chad {
         return 850;
     }
 
+    /** 
+     * Initialize Chad's slingshot and sword.
+     */
+    initWeapons() {
```

Comment:
Thanks for making the initWeapons method. I'll come in handy for instantiating the sword and slingshot.